test/integration/test_membership.c

```diff
@@ -111,14 +111,7 @@ TEST(membership, join, setUp, tearDown, 0, NULL)
 	PREPARE("INSERT INTO test(n) VALUES(1)", &stmt_id);
 	EXEC(stmt_id, &last_insert_id, &rows_affected);
 
-	/* The table is visible from the new node */
```

Comment:
Can we try to elect node 2 here, so that this functionality is still tested? Or is it tested elsewhere?

Reply:
Yes, I just did the simplest thing here, but modifying it to elect and talk to node 2 should be straightforward.